### PR TITLE
Buffs Void heretic carving knife into a good throw weapon

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request

Buffs carving knife into a good throw weapon, 5 damage less than cult dagger. It now has slightly higher embed chance than a combat knife and can become quite a pain to deal with. I believe that its worth it for an item quite far in the research tree.  

## Why It's Good For The Game

Fills in a niche in heresy for throwable weapons.

## Changelog
:cl: Sparkezelpl
balance: Carving knife now deals more damage on throw and can embed in your enemies.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
this should probably be mentioned in an in-game description somewhere.

**Comment by Sparkezel** [comment]
Sure that would be a good idea! Please wait a moment

File: code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm [hunk 1]
```diff
@@ -99,7 +99,7 @@
 /datum/eldritch_knowledge/rune_carver
 	name = "Carving Knife"
 	gain_text = "Etched, carved... eternal. I can carve the monolith and evoke their powers!"
-	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them, to create the carving knife transmute a knife with a glass shard and a piece of paper."
+	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them. You can also use them as a good throw weapon, to create the carving knife transmute a knife with a glass shard and a piece of paper."
```
Comment (LemonInTheDark) [inline code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm:102]: throwing != throw, I personally think the current structure is a bit off, but that part's up to you
```suggestion
	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them. They make quite a handy throwing weapon. To create the carving knife transmute a knife with a glass shard and a piece of paper."
```

Comment (LemonInTheDark) [reply code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm:102]: also use better commit names

Comment (Sparkezel) [reply code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm]: alright, I'm still quite new to all of this 🙂

Comment (LemonInTheDark) [reply code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm]: it's all good, we're all new deep down

**Comment by LemonInTheDark** [comment]
Could you update the changelog to be a bit more clear? Don't give stats or anything, something like the void carving knife now does more damage on throw/it embeds.

**Pull request merged**